### PR TITLE
Add virtualbox create/delete scripts

### PR DESCRIPTION
**Pull request by dpb587-pivotal**
To avoid copy/pasting the exact same commands every time to get a local director running.

### CODE REVIEW

File: virtualbox/create-env.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eu
```
Comment (cppforlife) [inline virtualbox/create-env.sh]: should we add -x ?

File: virtualbox/create-env.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+echo
+echo
+echo "===> Creating BOSH Director ===="
+echo
+
+bosh create-env bosh-deployment/bosh.yml \
+  --state state.json \
+  --ops-file bosh-deployment/virtualbox/cpi.yml \
+  --ops-file bosh-deployment/virtualbox/outbound-network.yml \
+  --ops-file bosh-deployment/bosh-lite.yml \
```
Comment (cppforlife) [inline virtualbox/create-env.sh]: let's throw in uaa and credhub in here.

File: virtualbox/create-env.sh (hunk 1 above)
Comment (dpb587-pivotal) [reply virtualbox/create-env.sh]: no; this should be as human-friendly as we can make it. if they're curious, they should look at the script

File: virtualbox/create-env.sh (hunk 2 above)
Comment (dpb587-pivotal) [reply virtualbox/create-env.sh]: okay

**Pull request merged**